src/main/java/com/upserve/uppend/lookup/LookupData.java

```diff
@@ -38,7 +38,7 @@
 
     // Timestamped references for readers
     private final AtomicStampedReference<LookupMetadata> timeStampedMetadata;
-    private final AtomicInteger reloadStamp;
+    /*private final*/ AtomicInteger reloadStamp;  // removed 'private final' to support unit testing
```

Comment:
Removing private is fine - why do we need to remove final?
Can't we just set the value in the test?